lib/fizzy/execute.cpp

```diff
@@ -1473,6 +1479,46 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             }
             break;
         }
+        case Instr::f32_convert_i32_s:
+        {
+            convert<int32_t, float>(stack);
+            break;
+        }
+        case Instr::f32_convert_i32_u:
+        {
+            convert<uint32_t, float>(stack);
+            break;
+        }
+        case Instr::f32_convert_i64_s:
+        {
+            convert<int64_t, float>(stack);
+            break;
+        }
+        case Instr::f32_convert_i64_u:
+        {
+            convert<uint64_t, float>(stack);
+            break;
+        }
+        case Instr::f64_convert_i32_s:
+        {
+            convert<int32_t, double>(stack);
+            break;
+        }
+        case Instr::f64_convert_i32_u:
+        {
+            convert<uint32_t, double>(stack);
+            break;
+        }
+        case Instr::f64_convert_i64_s:
+        {
+            convert<int64_t, double>(stack);
+            break;
+        }
+        case Instr::f64_convert_i64_u:
+        {
+            convert<uint64_t, double>(stack);
+            break;
+        }
         case Instr::f32_add:
```

Comment:
Hm, these seem to be out of order. We try to maintain the opcode layout in the switch statement.

Reply:
Fixed.